src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -2,256 +2,27 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Buffers;
-using System.Buffers.Binary;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
-using System.Globalization;
 using System.IO;
 using System.Text;
-using System.Threading;
 using System.Security;
 
 namespace System
 {
     public sealed partial class TimeZoneInfo
     {
-        private const string DefaultTimeZoneDirectory = "/usr/share/zoneinfo/";
-        private const string ZoneTabFileName = "zone.tab";
-        private const string TimeZoneEnvironmentVariable = "TZ";
-        private const string TimeZoneDirectoryEnvironmentVariable = "TZDIR";
+        private const string _timeZoneFileName = "zone.tab";
```

Comment:
const variables should be cased like they were previously - Pascal cased. `_camelCase` is for fields.

[https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md)

Reply:
Thanks for the catch, I didn't know that rule 12 trumped rule 3 :)